src/components/DownloadToast/index.tsx

```diff
@@ -32,30 +33,15 @@ function getETAStringFromMs(etaInMs: number) {
   const minutes = Math.floor((totalSeconds % 3600) / 60)
   const hours = Math.floor((totalSeconds % 86400) / 3600)
   const days = Math.floor(totalSeconds / 86400)
-  let etaString = ''
-  if (days !== 0) etaString += `${days}d`
-  if (hours !== 0) etaString += (etaString === '' ? '' : ':') + `${hours}h`
-  if (minutes !== 0) etaString += (etaString === '' ? '' : ':') + `${minutes}m`
-  if (seconds !== 0) etaString += (etaString === '' ? '' : ':') + `${seconds}s`
-  if (etaString === '') return '0s'
-  return etaString
-}
 
-function getSizeStringFromBytes(bytes: number) {
-  const kb = bytes / 1024
-  const mb = kb / 1024
-  const gb = mb / 1024
-  const tb = gb / 1024
-  if (tb > 1) {
-    return `${Math.floor(tb)}TB`
-  } else if (gb > 1) {
-    return `${Math.floor(gb)}GB`
-  } else if (mb > 1) {
-    return `${Math.floor(mb)}MB`
-  } else if (kb > 1) {
-    return `${Math.floor(kb)}KB`
+  if (days > 0) {
+    return `${days}d`
+  } else if (hours > 0) {
+    return `${hours}h:${minutes}m`
+  } else if (minutes > 0) {
+    return `${minutes}m:${seconds}s`
   } else {
-    return `0MB`
+    return `${seconds}s`
   }
```

Comment:
Would be nice if we could put this in `@hyperplay/utils` but completely understand if you don't have time right now. I think there are a couple other places we show an ETA string on the frontend and we could reuse that function in those components as well.

Reply:
Just moved this method there and added a couple of others we use in the client.
Wrote tests for all of them as well.